hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/shell/TestOzoneShellHA.java

```diff
@@ -2190,8 +2195,7 @@ public void testClientBucketLayoutValidation() {
     ParameterException exception = assertThrows(ParameterException.class,
         () -> execute(ozoneShell, arg2));
     assertThat(exception.getMessage())
-        .contains("expected one of [FILE_SYSTEM_OPTIMIZED, OBJECT_STORE, " +
-            "LEGACY]");
+        .contains("cannot convert '' to BucketLayout");
```

Comment:
Should it not be "Unknown bucket layout: " as per [this](https://github.com/apache/ozone/blob/cfe56dee0e85ed7cd1efa7edba72228668c2e292/hadoop-ozone/cli-shell/src/main/java/org/apache/hadoop/ozone/shell/bucket/CreateBucketHandler.java#L79C45-L79C67)

Reply:
The complete error message is this:

```
Invalid value for option '--layout': cannot convert '' to BucketLayout (java.lang.IllegalArgumentException: Unknown bucket layout: )
```